### PR TITLE
Another egotrip bless

### PR DESCRIPTION
**Pull request by Durhamite2118**
Looking to add one more.  Replaced Angus.

### CODE REVIEW

File: code/code/cmd/cmd_egotrip.cc [hunk 1]
```diff
@@ -88,11 +88,12 @@ std::map <spellNumT,ego_imm_blessing> init_ego_imm_blessing()
         AFFECT_DASH_BLESSING,
         APPLY_FOC,
         "<B>reflection<1>"); // Steal my blessing again, and I'll break you. -D
-  blessings[AFFECT_ANGUS_BLESSING]=
-    ego_imm_blessing("Angus",
-        AFFECT_ANGUS_BLESSING,
-        APPLY_WIS,
-        "<g>wisdom<1>");
+  blessings[AFFECT_SIDARTHA_BLESSING]=
+    ego_imm_blessing("SIDARTHA",
```
Comment (cizra) [inline code/code/cmd/cmd_egotrip.cc:92]: Why's this all caps?

**Review by cizra — CHANGES_REQUESTED** [review]

**Review by cizra — APPROVED** [review]

File: code/code/cmd/cmd_egotrip.cc [hunk 2]
```diff
@@ -88,11 +88,12 @@ std::map <spellNumT,ego_imm_blessing> init_ego_imm_blessing()
...
-  blessings[AFFECT_ANGUS_BLESSING]=
-    ego_imm_blessing("Angus",
-        AFFECT_ANGUS_BLESSING,
-        APPLY_WIS,
-        "<g>wisdom<1>");
+  blessings[AFFECT_SIDARTHA_BLESSING]=
+    ego_imm_blessing("Sidartha",
+        AFFECT_SIDARTHA_BLESSING,
+        APPLY_SPE,
+	APPLY_HASTE,
```
Comment (wershlak) [inline code/code/cmd/cmd_egotrip.cc:95]: Does this work?

File: code/code/spec/spec_objs.cc [hunk 3]
```diff
@@ -6785,7 +6785,7 @@ extern int livingVines(TBeing *vict, cmdTypeT cmd, const char *, TObj *o, TObj *
 extern int dkSword(TBeing *vict, cmdTypeT cmd, const char *, TObj *o, TObj *);
 extern int moltenWeapon(TBeing *vict, cmdTypeT cmd, const char *arg, TObj *o, TObj *);
 extern int glacialWeapon(TBeing *vict, cmdTypeT cmd, const char *arg, TObj *o, TObj *);
-
+extern int lightningboltWeapon(TBeing *vict, cmdTypeT cmd, const char *arg, TOb;
```
Comment (cizra) [inline code/code/spec/spec_objs.cc:6788]: Whoa, you're now mixing up egotrip and lightning bolt. That's no bueno.

**Review by cizra — CHANGES_REQUESTED** [review]

**Review by Durhamite2118 — COMMENTED** [review]
Deleted lightproc from egotrip bless.

File: code/code/cmd/cmd_egotrip.cc (hunk 2 above)
Comment (Durhamite2118) [reply code/code/cmd/cmd_egotrip.cc:95]: Works from my view unless Elmo has other concerns.

File: code/code/spec/spec_objs.cc (hunk 3 above)
Comment (Durhamite2118) [reply code/code/spec/spec_objs.cc]: See below

File: code/code/cmd/cmd_egotrip.cc (hunk 2 above)
Comment (wershlak) [reply code/code/cmd/cmd_egotrip.cc:95]: What I mean is APPLY_HASTE isn't a thing defined in the code. APPLY_SPE is defined here: https://github.com/sneezymud/sneezymud/blob/master/code/code/misc/enum.h#L333-L372

This code won't compile

Comment (Durhamite2118) [reply code/code/cmd/cmd_egotrip.cc]: Added hitroll instead.

**Review by wershlak — CHANGES_REQUESTED** [review]
You also need to delete these two lines: https://github.com/sneezymud/sneezymud/blob/master/code/code/sys/handler.cc#L872-L873

File: code/code/cmd/cmd_egotrip.cc [hunk 4]
```diff
@@ -88,11 +88,14 @@ std::map <spellNumT,ego_imm_blessing> init_ego_imm_blessing()
...
-  blessings[AFFECT_ANGUS_BLESSING]=
-    ego_imm_blessing("Angus",
-        AFFECT_ANGUS_BLESSING,
-        APPLY_WIS,
-        "<g>wisdom<1>");
+  blessings[AFFECT_SIDARTHA_BLESSING]=
+    ego_imm_blessing("Sidartha",
+        AFFECT_SIDARTHA_BLESSING,
+        APPLY_SPE,
+	"<b>the mongoose<1>");
```
[the remark this answers is not in the document]
Comment (Durhamite2118) [reply code/code/cmd/cmd_egotrip.cc]: As requested.

[the remark this answers is not in the document]
Comment (Durhamite2118) [reply code/code/cmd/cmd_egotrip.cc]: done and done.

**Comment by wershlak** [comment]
This seems ok now. Thanks for fixing things.

**Pull request merged**